cds-snc/c19-benefits-node
# Fix the issue with hitting back and receiving multiple benefits

## Description

Add a pruneSessionData method to the session.helpers.js that allows us to remove form data from the session.

This fixes the bug where the user uses the back key to navigate back to see what a different path would return and we ended up with multiple benefits. 

Also adds a test that duplicates this bug.

Closes #378 

## To Test

Navigate around the site using the back button to go back to some forks in the flow and visit down those.

You should only see benefits that are relevant to the fork you are on.

Figma is really useful for visualizing the different paths.
